deploy/ocs-operator/manifests/provider-role.yaml

```diff
@@ -16,6 +16,7 @@ rules:
     resources:
       - cephfilesystemsubvolumegroups
       - cephblockpoolradosnamespaces
+      - cephblockpools
     verbs:
       - get
       - list
```

Comment:
For the Ceph block pool, you only need to get access. I am not sure Is it fair to add a new entry for the same API group?

Reply:
We need list access as we are listing the blockpools with the refactored code

Reply:
There is a need for both get and list access for ceph block pools